docs/deployment.md

```diff
@@ -87,6 +87,7 @@ Options:
                                   [default: TLSv1]
   --header TEXT                   Specify custom default HTTP response headers
                                   as a Name:Value pair
+  --app-dir TEXT                  Look for APP in the specified directory. Defaults to '.'
```

Comment:
What do we think about changing this to "Defaults to the current working directory" ?

Reply:
Yes "Defaults to the current working directory" looks good.